website/src/content/docs/reference/configuration.mdx

```diff
@@ -451,6 +451,18 @@ How many characters can be written on a single line.
 
 > Default: `80`
 
+### `formatter.bracketSameLine`
+
+Choose whether the ending `>` of a multi-line HTML ( or HTML-like ) element should be on the last attribute line or not
+
+> Default: false
+
+### `formatter.bracketSpacing`
+
+Choose wether spaces should be added between brackets and inner values
```

Comment:
```suggestion
Choose whether spaces should be added between brackets and inner values
```

Reply:
I fixed the remaining typos, now should be fine